### PR TITLE
Add docs for example IAM app

### PR DESCRIPTION
**Pull request by mikkeloscar**

### CODE REVIEW

File: docs/user-guide/example-credentials.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      volumeMounts:
+        - name: credentials
+          mountPath: /meta/credentials
+          readOnly: false
+
+The ``gerry`` *sidecar* container mounts the shared ``credentials`` mount point
+under ``/meta/credentials`` and writes the credential files ``user.json`` and
+``client.json`` to this location.
+
+To read these files from the ``myapp`` container, the shared ``credentails``
```
Comment (linki) [inline docs/user-guide/example-credentials.rst]: `credentails` typo

**Comment by linki** [comment]
thanks for that

**Comment by linki** [comment]
👍

File: docs/user-guide/example-credentials.rst (hunk 1 above)
Comment (mikkeloscar) [inline docs/user-guide/example-credentials.rst]: Thanks!

**Comment by mikkeloscar** [comment]
:+1:

**Comment by linki** [comment]
👍

**Pull request merged**